common/datavalues/Cargo.toml

```diff
@@ -16,7 +16,7 @@ common-exception = {path = "../exception"}
 # Crates.io dependencies
 anyhow = "1.0.41"
 paste = "^1.0"
-ordered-float = "2.0"
+ordered-float = "2.5"
```

Comment:
Hmm, why we bump ordered-float here?

Reply:
I see.
This is scanned by https://deps.rs/repo/github/datafuselabs/datafuse
It seems that's the dep bot job, don't know why it's not working